lib/elixir/src/elixir_parser.yrl

```diff
@@ -294,19 +294,23 @@ stab -> stab eoe stab_expr : ['$3'|'$1'].
 stab_eoe -> stab : '$1'.
 stab_eoe -> stab eoe : '$1'.
 
-stab_expr -> expr : '$1'.
-stab_expr -> stab_op_eol stab_maybe_expr : build_op('$1', [], '$2').
-stab_expr -> empty_paren stab_op_eol stab_maybe_expr :
-               build_op('$2', [], '$3').
-stab_expr -> call_args_no_parens_all stab_op_eol stab_maybe_expr :
-               build_op('$2', unwrap_when(unwrap_splice('$1')), '$3').
-stab_expr -> stab_parens_many stab_op_eol stab_maybe_expr :
-               build_op('$2', unwrap_splice('$1'), '$3').
-stab_expr -> stab_parens_many when_op expr stab_op_eol stab_maybe_expr :
-               build_op('$4', [{'when', meta_from_token('$2'), unwrap_splice('$1') ++ ['$3']}], '$5').
-
-stab_maybe_expr -> 'expr' : '$1'.
-stab_maybe_expr -> '$empty' : nil.
+%% Here, `element(1, Token)` is the stab operator, while `element(2, Token)` is
+%% the expression.
+stab_expr -> expr :
+               '$1'.
+stab_expr -> stab_op_eol_and_maybe_expr :
+               build_op(element(1, '$1'), [], element(2, '$1')).
```

Comment:
I would agree with that but we would still have to leave `element(2` for the expression right? So maybe it makes sense to leave both as is. Wdyt?